content/dstu2/condition.md

```diff
@@ -166,5 +192,5 @@ Note: abatementDateTime is not supported for diagnosis
 [category]: http://hl7.org/fhir/DSTU2/valueset-condition-category.html
 [clinical status]: http://hl7.org/fhir/DSTU2/valueset-condition-clinical.html
 [condition-category]: http://hl7.org/fhir/condition-category
-[argonaut extension-codes]: http://argonaut.hl7.org/extension-codes
+[argonaut]: http://argonaut.hl7.org
```

Comment:
This link doesn't go anywhere, is it still worth displaying it as a hyperlink in the note? I know the previous one didn't lead anywhere either but as a consumer if I see a hyperlink I would expect to be able to click it to read more.